src/motion/features/layout/Measure.tsx

```diff
@@ -28,6 +27,16 @@ class Measure extends React.Component<SyncProps> {
         isSharedLayout(syncLayout) && syncLayout.register(visualElement)
         isSharedLayout(framerSyncLayout) &&
             framerSyncLayout.register(visualElement)
+
+        visualElement.onUnmount(() => {
+            if (isSharedLayout(syncLayout)) {
+                syncLayout.remove(visualElement as any)
+            }
+
+            if (isSharedLayout(framerSyncLayout)) {
+                framerSyncLayout.remove(visualElement as any)
+            }
+        })
```

Comment:
Same here, why is this change required?

Reply:
Same as above really, this ones just as bad really as these contexts call `createBatcher` by default. So it's not just a matter of bundling in empty contexts, the batcher also gets included in the minimal bundle.

Reply:
Gotcha 👌